src/wiktextract/extractor/es/pronunciation.py

```diff
@@ -41,7 +41,19 @@ def process_pron_graf_template(
     )
 
     for sound_keys in sound_keys_grouped.values():
-        sound = Sound()
+        # variations is used to separate out arguments with different
+        # numbers at the *end*; Spanish pron templates can have several
+        # layers of arguments, with numbers at both sides. Here, we
+        # create minor variations, for example when a pronunciation has
+        # two sets of audio-files, like es-wiktionary 'brown': the
+        # UK and US files should not be in the same 'audio' field,
+        # because the field name is meaningful and should contain only
+        # one item. It makes some sense to separate these arguments
+        # out into their own variant Sound objects that then get
+        # filled with all the missing fields from their main (dict
+        # key 0) Sound object that got all the arguments without
+        # postpositional numbers.
+        variations: dict[int, Sound] = defaultdict(Sound)
```

Comment:
I tried to keep the number changes small. We can always make it better later.

I took a look at how variations is used, and in retrospect it's a bit weird... It works, because we just create a reference to a defaultdict, and so there is not need to touch "sound" at all. We want to modify the underlying Sound object, so we can't create and assign the variation later (because of the looping structure, we look at keys and assign them in the correct Sound object)... I'm not actually sure how to do this "properly". I'll add a comment, feel free to modify this if you feel like it. I just don't know what would be the best here.